packages/static-build/src/utils/build-output-v2.ts

```diff
@@ -0,0 +1,125 @@
+import path from 'path';
+import { pathExists } from 'fs-extra';
+import { promises as fs } from 'fs';
```

Comment:
```suggestion
import { stat, pathExists } from 'fs-extra';
```

Just use `stat()` from fs-extra.